### PR TITLE
ONTO-63 fix order & formatting issues.

### PR DESCRIPTION
**Pull request by hrehman-sphereon**
React Native app was not working from my side due lack of installations.
ReadMe is funcationally correct though.

### CODE REVIEW

**Comment by nklomp** [comment]
Guys I need this merged and handed over before 16:15 today!

Please resolve the merge conflict. @hrehman-sphereon @maikel-maas

File: README.md [hunk 1]
```diff
@@ -4,28 +4,39 @@ ONTOCHAIN Demo
...
+
 ###Setup
+
+#### Build & start
+
+clone the repository:
+
+```bash
+cd <workspace>
+git clone git@github.com:Sphereon/onto-web-demo.git
```
Comment (mehmetbicak) [inline README.md:25]: Could not find this repo (git@github.com:Sphereon/onto-web-demo.git). 
Should it be  ssh: git@github.com:Gimly-Blockchain/did-siop-auth-web-demo.git
or https: https://github.com/Gimly-Blockchain/did-siop-auth-web-demo.git ?

File: README.md [hunk 2]
```diff
@@ -36,56 +47,83 @@ RP_PRIVATE_HEX_KEY=850e54b92c6291a1ff7b8c3ef30e032571ed77c9e5c78b1cd6ee5fec4fea9
...
-clone the repository:
+From the root directory
+
 ```bash
-git clone git@github.com:Sphereon/onto-web-demo.git
+cd <workspace>/onto-web-demo
+yarn global add concurrently
+yarn global add ts-node
+yarn install-all
+yarn build-types
```
Comment (mehmetbicak) [inline README.md:52]: yarn build-types gives me this error : 

src/auth-model.ts:21:10 - error TS2339: Property 'userName' does not exist on type 'AuthResponse'.

21     this.userName = userName;
            ~~~~~~~~

**Review by zoemaas — APPROVED** [review]
It looks good to me

File: README.md [hunk 3]
```diff
@@ -36,56 +47,83 @@ RP_PRIVATE_HEX_KEY=850e54b92c6291a1ff7b8c3ef30e032571ed77c9e5c78b1cd6ee5fec4fea9
...
-The server will start on port 5001, the client will start & open a browser on http://localhost:3000/
+The server will start on port `5001`, the client will start & open a browser on `http://localhost:3000/`
 
 #### Docker
+
 From the root folder run:
+
 ```bash
+cd <workspace>/onto-web-demo
 docker build -t onto-web-demo .
```
Comment (mehmetbicak) [inline README.md:59]: During docker build -- Getting the same error on build-types

 => ERROR [6/6] RUN yarn build-types                                                                                                                            2.6s
------
 > [6/6] RUN yarn build-types:
#9 0.768 yarn run v1.22.5
#9 0.795 $ cd onto-demo-shared-types && yarn run build
#9 0.929 $ tsc -b
#9 2.465 src/auth-model.ts(21,10): error TS2339: Property 'userName' does not exist on type 'AuthResponse'.

**Review by mehmetbicak — COMMENTED** [review]
3 comments related to repo address and build errors

**Pull request merged**